From aace9b1675819feab81d50f694391c7d854d9822 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Fri, 16 Nov 2018 12:50:53 +0100
Subject: actually initialize kcrash for xclipboardsync

Summary:
to successfully use kcrash when linking with as-needed (which is a default
flag on many linux distros) one also needs to call KCrash::initialize.

https://markmail.org/thread/zv5pheijaze72bzs

Test Plan: builds; correctly links kcrash

Reviewers: davidedmundson

Reviewed By: davidedmundson

Subscribers: kwin

Tags: #kwin

Differential Revision: https://phabricator.kde.org/D16923
---
 helpers/xclipboardsync/main.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/helpers/xclipboardsync/main.cpp b/helpers/xclipboardsync/main.cpp
index 2f06e9c..1a5e5ee 100644
--- a/helpers/xclipboardsync/main.cpp
+++ b/helpers/xclipboardsync/main.cpp
@@ -19,6 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *********************************************************************/
 #include "waylandclipboard.h"
 
+#include <KCrash>
 #include <QGuiApplication>
 
 #include <config-kwin.h>
@@ -40,6 +41,7 @@ int main(int argc, char *argv[])
                 argv[0], qPrintable(app.platformName()));
         return 1;
     }
+    KCrash::initialize();
     new WaylandClipboard(&app);
     return app.exec();
 }
-- 
cgit v0.11.2
